docs/getting-started/reading-and-writing.mdx

```diff
@@ -108,4 +108,28 @@ async function getUser(username) {
 
     return result.data;
 }
+```
+
```

Comment:
As covered in the last review, you need to move the note on private mode to `getting-started/read-access-control`. This is the wrong section.